src/lib/components/ScriptureViewSofria.svelte

```diff
@@ -441,12 +448,9 @@ TODO:
                                     workspace.paragraphDiv = document.createElement('div');
                                     workspace.paragraphDiv.classList.add(paraClass);
                                 } else if (sequenceType == 'introduction') {
+                                    workspace.phraseDiv = startPhrase(workspace, 'keep');
```

Comment:
What does this do?

Reply:
Primarily, the change moved this to before creating a new paragraph div.  Didn't test this out enough with introductions.  Overall, what the start phrase does is move any existing phrase i (containing the last phrase in the previous text message) into the paragraph div.  The keep means we don't change the phrase index (the a b c d) which isn't needed for introductions.  Prior to the change, the introduction text was showing up in the paragraph following the one it should have.  Most things were displaced by one paragraph which caused some crazy formatting in the glossary.